generic/system/ksp-prevent-crypto-miners.yaml

```diff
@@ -0,0 +1,68 @@
+# KubeArmor is an open source software that enables you to protect your cloud workload at run-time.
+# To learn more about KubeArmor visit: 
+# https://www.accuknox.com/kubearmor/ 
+
+apiVersion: security.kubearmor.com/v1
+kind: KubeArmorPolicy
+metadata:
+  name: prevent-crypto-miners
```

Comment:
```suggestion
  name: ksp-mitre-prevent-crypto-miners
```

Reply:
As you have seen other policy file references, the policy name in Metadata starts with ksp and its appropriate mitre tag. 
I suggest to follow the same.

Reply:
Changed as suggested